src/features/components/molecules/slide-show/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect, ReactElement } from 'react'
+import * as styles from './slide-show.module.scss'
+import dclsx from 'features/utils/dclsx'
+
+type SlideshowItemProps = {
+    key: string
+    image: ReactElement
+}
+type SlideshowItemArrayProps = {
+    items: SlideshowItemProps[]
+}
+
+const Slideshow = ({ items }: SlideshowItemArrayProps) => {
+    const [currentImageIndex, setCurrentImageIndex] = useState(0)
```

Comment:
```suggestion
    const [current_image_index, setCurrentImageIndex] = useState(0)
```

Reply:
Thanks @prince-deriv for suggesting 🙏 
Just asking what is the difference ? 👀